test/e2e/serial/workload_placement_test.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package serial
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	nrtv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+
+	nropv1alpha1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1alpha1"
+	"github.com/openshift-kni/numaresources-operator/pkg/machineconfigpools"
+	e2efixture "github.com/openshift-kni/numaresources-operator/test/utils/fixture"
+	e2enrt "github.com/openshift-kni/numaresources-operator/test/utils/noderesourcetopologies"
+	"github.com/openshift-kni/numaresources-operator/test/utils/nrosched"
+)
+
+const NROperatorObjectName = "numaresourcesoperator"
+
+var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
+	var testFixture *e2efixture.Fixture
+	var nroSchedObj *nropv1alpha1.NUMAResourcesScheduler
+	var schedulerName string
+	var nodeResourceTopologyList nrtv1alpha1.NodeResourceTopologyList
+	var nodeResourceTopologies []nrtv1alpha1.NodeResourceTopology
+
+	BeforeEach(func() {
+		var err error
+
+		By("Setup namespaces")
+		testFixture, err = e2efixture.Setup("e2e-test-workload-placement")
+		Expect(err).NotTo(HaveOccurred(), "unable to setup test fixture")
+
+		By("Check for NumaResourceScheduler")
+		nroSchedObj = &nropv1alpha1.NUMAResourcesScheduler{}
+		nroSchedObjKey := client.ObjectKey{Name: nrosched.NROSchedObjectName}
+		err = testFixture.Client.Get(context.TODO(), nroSchedObjKey, nroSchedObj)
+		Expect(err).NotTo(HaveOccurred(), "cannot get %q scheduler from the cluster", nrosched.NROSchedObjectName)
+
+		schedulerName = nroSchedObj.Status.SchedulerName
+		Expect(schedulerName).NotTo(BeEmpty(), "cannot autodetect TAS name from the NRO Scheduler object")
+
+		By("Check worker nodes topology is ok to run the tests")
+		err = testFixture.Client.List(context.TODO(), &nodeResourceTopologyList)
+		Expect(err).NotTo(HaveOccurred(), "unable to find NodeResourceTopolyList in the cluster")
+
+		topoloyManagerPolicy := nrtv1alpha1.SingleNUMANodeContainerLevel
+		nodeResourceTopologies = e2enrt.FilterTopologyManagerPolicy(nodeResourceTopologyList.Items, topoloyManagerPolicy)
+		for _, nrt := range nodeResourceTopologies {
+			fmt.Printf("jlom NRT name %#v\n", nrt.Name)
```

Comment:
`jlom`? you can use `%s` to format strings. And you probably want to use `klog` rather than `fmt`

Reply:
yep, forgot to delete this "debug" lines before commit.

Reply:
Debug per se is fine. Let's just use `klog`